### PR TITLE
[ubuntu-dev] Add web-terminal

### PR DESCRIPTION
**Pull request by nt1m**

### CODE REVIEW

**Comment by bnjbvr** [comment]
(as indicated on IRC, I'm clueless before this change, so implicitly un-assigning myself by unsubscribing to notifications)

File: ubuntu-dev/ubuntu-dev.dockerfile [hunk 1]
```diff
@@ -168,6 +168,9 @@ RUN mkdir /home/user/.phacility \
  && echo "\n# Phabricator helper." >> /home/user/.bashrc \
  && echo "PATH=\"\$PATH:/home/user/.phacility/arcanist/bin\"" >> /home/user/.bashrc
 
+# Install web-terminal
```
Comment (jankeromnes) [inline ubuntu-dev/ubuntu-dev.dockerfile]: Nit: + "the latest" and + ".".

**Comment by nt1m** [comment]
> Why did you choose it among all the web terminals out there? 
I tried many different terminals, and this one seems like the lightest. The UX is not the best, but it's definitively enough for a quick terminal in the contributions page (I have a working redesign without bootstrap locally btw!).

> did you test it locally, in order to make sure that what you're seeing up here will work well for all Janitor projects?
Yep :)

**Comment by nt1m** [comment]
Thanks for the review btw :)

**Pull request merged**